apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/ChatView.kt

```diff
@@ -1497,6 +1541,78 @@ private fun TopEndFloatingButton(
   }
 }
 
+@Composable
+private fun TopCenterFloatingButton(
```

Comment:
it's not a button. It's maybe FloatingDate but for sure not a button

Reply:
and since we have only one floatingDate, there is no need to call it `TopCenter`. For floating buttons it makes sense because there are two buttons in ChatView